pageserver/src/layered_repository/storage_layer.rs

```diff
@@ -100,6 +100,11 @@ pub trait Layer: Send + Sync {
     /// - An image layer represents snapshot at one LSN, so end_lsn is always the snapshot LSN + 1
     fn get_lsn_range(&self) -> Range<Lsn>;
 
+    /// Get layer size
+    fn size(&self) -> Result<u64> {
+        Ok(std::fs::metadata(self.local_path().unwrap())?.len())
+    }
+
```

Comment:
I several times before needed function which calculates layers size (for example for choosing moment for layer compaction)
and have to write this code once and once again. I think that it will be useful to have this function, although right now it is not needed.